src/Quadrature.h

```diff
@@ -178,12 +182,20 @@ class LeonardPolarQuad: public Quadrature {
 class GLPolarQuad: public Quadrature {
 
 private:
+  /** the roots to the Legendre polynomial of degree _num_polar */
+  std::vector <double> _roots;
```

Comment:
I don't think we can get the exacat Gauss Legendre abscissae from _thetas since the thetas are adajusted. 
It does incorporate weight correction. If we only use the corrected weights, (which converge k-eff better and likely yield a more accurate answer) then there's no need for the class variable